examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/weight_only/run_clm_no_trainer.py

```diff
@@ -403,11 +403,12 @@ def calib_func(prepared_model):
             max_seq_length=args.gptq_max_seq_length,
         )
         dataloader_for_calibration = dataloaderPreprocessor.get_prepared_dataloader()
-        from neural_compressor.torch.algorithms.weight_only.utility import move_input_to_device
+        from neural_compressor.torch.utils import get_model_device, move_input_device
```

Comment:
I saw this move_input_to_device function implemented/copied in multiple location e.g.. weight-only/smoothquant.  it's better to be replaced with this single mov_input_device function under general utily (mayber after this release)

Reply:
@Kaihui-intel FYI. I had moved these functions to general utils.